core/runtime/src/main/java/org/eclipse/sw360/antenna/workflow/AntennaWorkflow.java

```diff
@@ -75,6 +82,27 @@ public AntennaWorkflow(AntennaWorkflowConfiguration antennaWFConfig) {
         }
     }
 
+    private String makeStringForFailCausingResults(Map<String, Set<IEvaluationResult>> failCausingResults) {
+        StringBuilder failCausingResultMessage = new StringBuilder("Build fails due to fail causing results \n");
+
+        failCausingResults.forEach((workflowItemName, evaluationResults) ->
+                failCausingResultMessage.append(workflowItemName)
+                        .append(" failed with evaluation results: \n")
+                        .append(makeStringForEvaluationResults(evaluationResults))
```

Comment:
I would prefer to not have a StringBuilder in a StringBuilder, we could basically add the iteration from the called method in here without a to complex method, I think